osmaxx-py/excerptconverter/dummyexcerptconverter/dummy_excerpt_converter.py

```diff
@@ -93,29 +95,32 @@ def execute_task(extraction_order_id, supported_export_formats, execution_config
                 if wait_time > 30:
                     raise
 
-        fake_work_waiting_time_in_seconds = 5
+        try:
+            converter_helper = ConverterHelper(extraction_order)
+            fake_work_waiting_time_in_seconds = 5
 
-        # now set the new state
-        extraction_order.state = models.ExtractionOrderState.WAITING
-        extraction_order.save()
+            # now set the new state
+            extraction_order.state = models.ExtractionOrderState.WAITING
+            extraction_order.save()
 
-        time.sleep(fake_work_waiting_time_in_seconds)
+            time.sleep(fake_work_waiting_time_in_seconds)
 
-        # now set the new state
-        extraction_order.state = models.ExtractionOrderState.PROCESSING
-        extraction_order.save()
+            # now set the new state
+            extraction_order.state = models.ExtractionOrderState.PROCESSING
+            extraction_order.save()
 
-        message_text = _('Your extraction order "%s" has been started' % extraction_order)
-        BaseExcerptConverter.inform_user(extraction_order.orderer, messages.INFO, message_text, False)
+            message_text = _('The Dummy conversion of extraction order "%s" has been started.') % extraction_order.id
+            converter_helper.inform_user(messages.INFO, message_text, False)
 
-        DummyExcerptConverter.create_output_files(execution_configuration, extraction_order, supported_export_formats)
-
-        time.sleep(fake_work_waiting_time_in_seconds)
+            DummyExcerptConverter.create_output_files(
+                execution_configuration,
+                extraction_order,
+                supported_export_formats, converter_helper
+            )
 
-        # now set the new state
-        extraction_order.state = models.ExtractionOrderState.FINISHED
-        extraction_order.save()
+            time.sleep(fake_work_waiting_time_in_seconds)
 
-        # inform the user of the status change.
-        message_text = _('Your extraction order %s has been processed' % extraction_order)
-        BaseExcerptConverter.inform_user(extraction_order.orderer, messages.SUCCESS, message_text)
+            # now set the new state (if all files have been processed) and inform the user about the state
+            converter_helper.file_conversion_finished()
+        except:
+            converter_helper.inform_user(messages.ERROR, traceback.format_tb(sys.exc_info()), False)
```

Comment:
In production, don't show the trace to the user. Log it instead.

In development, consider raising the exception.